core/src/main/java/org/mule/runtime/core/internal/util/message/EventedResult.java

```diff
@@ -4,7 +4,7 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.runtime.module.extension.api.runtime.privileged;
+package org.mule.runtime.core.internal.util.message;
```

Comment:
This rename will be a problem I think, because [the mule-aggregators-module is using it](https://github.com/mulesoft/mule-aggregators-module/blob/main/src/main/java/org/mule/extension/aggregator/internal/privileged/CompletionCallbackWrapper.java#L49) (unlike its brother the `EventedSdkResult`).

Reply:
great catch! Damn evil twin!. Fixed